src/main/java/seedu/club/model/task/Description.java

```diff
@@ -0,0 +1,45 @@
+package seedu.club.model.task;
+
+//@@author yash-chowdhary
+
+import static java.util.Objects.requireNonNull;
+import static seedu.club.commons.util.AppUtil.checkArgument;
+
+/**
+ * Refers to the Description of a Task
+ */
+public class Description {
+
+    public static final String DESCRIPTION_VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    public static final String MESSAGE_DESCRIPTION_CONSTRAINTS = "Description should a non-empty alphanumeric string";
```

Comment:
Can description be more than alphanumeric? Eg some ppl use @ for at and ! to show importance and any other non-alphanumeric characters

Reply:
Hmm, I guess I could do that. However, this is visible to everyone, so I thought it should be in a proper format